src/main/java/com/github/jikoo/anvilunlocker/AnvilUnlocker.java

```diff
@@ -22,9 +22,10 @@ public void onEnable() {
 
 	@EventHandler
 	public void onInventoryOpen(InventoryOpenEvent event) {
+		int maximumCost = getConfig().getInt("maximumcost");
```

Comment:
Would prefer this to be inside the if statement - that way we aren't pulling a config value every time any inventory opens. It's a small optimization for sure, but an easy one.